doc/dependency_decisions.yml

```diff
@@ -1,172 +1,172 @@
 ---
 - - :permit
   - MIT
-  - :who: 
-    :why: 
+  - :who:
```

Comment:
if nothing in this file actually changed, can you revert changes here? want to keep unnecessary files out of the pr